Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -410,7 +429,7 @@ def audit_deps
           problem "Dependency '#{dep.name}' was renamed; use new name '#{dep_f.name}'."
         end
 
-        if @@aliases.include?(dep.name)
+        if @@aliases.include?(dep.name) && !dep_f.versioned_formula?
```

Comment:
Not sure why we'd want to drop this audit. I understand that someone running a tap might want to use an alias, but within the Homebrew organization, this seems like it should still be an audit failure.